lumigator/python/mzai/sdk/tests/int_test_datasets.py

```diff
@@ -27,51 +27,49 @@ def test_get_datasets_remote_ok(lumi_client):
+    datasets = lumi_client.datasets.get_datasets()
+    after = datasets.total
+    dataset = lumi_client.datasets.get_dataset(datasets.items[0].id)
+    assert dataset is not None
+    lumi_client.datasets.delete_dataset(datasets.items[0].id)
+    datasets = lumi_client.datasets.get_datasets()
+    assert after - before == 1
 
 
 def test_job_lifecycle_remote_ok(lumi_client, dialog_data):
     logger.info('Starting jobs lifecycle')
-    with Path.open(dialog_data) as file:
-        datasets = lumi_client.datasets.get_datasets()
-        if datasets.total > 0:
-            for remove_ds in datasets.items:
-                logger.info(f"Removing dataset {remove_ds.id}")
-                lumi_client.datasets.delete_dataset(remove_ds.id)
-        datasets = lumi_client.datasets.get_datasets()
-        before = datasets.total
-        dataset = lumi_client.datasets.create_dataset(dataset=file, format=DatasetFormat.JOB)
-        datasets = lumi_client.datasets.get_datasets()
-        after = datasets.total
-        assert after - before == 1
+    datasets = lumi_client.datasets.get_datasets()
+    if datasets.total > 0:
+        for remove_ds in datasets.items:
+            logger.info(f"Removing dataset {remove_ds.id}")
+            lumi_client.datasets.delete_dataset(remove_ds.id)
+    datasets = lumi_client.datasets.get_datasets()
+    before = datasets.total
+    dataset = lumi_client.datasets.create_dataset(dataset=dialog_data, format=DatasetFormat.JOB)
+    datasets = lumi_client.datasets.get_datasets()
+    after = datasets.total
+    assert after - before == 1
 
-        jobs = lumi_client.jobs.get_jobs()
-        assert jobs is not None
-        logger.info(lumi_client.datasets.get_dataset(dataset.id))
-        # job_create = JobCreate(name="test-job-int-001", model="hf://distilbert/distilbert-base-uncased", dataset=dataset.id)
-        job_create = JobCreate(name="test-job-int-001", model="hf://distilgpt2", dataset=dataset.id)
-        job_create.description = "This is a test job"
-        job_create.max_samples = 0
-        job_ret = lumi_client.jobs.create_job(JobType.EVALUATION, job_create)
-        assert job_ret is not None
-        jobs = lumi_client.jobs.get_jobs()
-        assert jobs is not None
-        assert len(jobs.items) == jobs.total
+    jobs = lumi_client.jobs.get_jobs()
+    assert jobs is not None
+    logger.info(lumi_client.datasets.get_dataset(dataset.id))
+    # job_create = JobCreate(name="test-job-int-001", model="hf://distilbert/distilbert-base-uncased", dataset=dataset.id)
+    job_create = JobCreate(name="test-job-int-001", model="hf://distilgpt2", dataset=dataset.id)
+    job_create.description = "This is a test job"
+    job_create.max_samples = 0
+    job_ret = lumi_client.jobs.create_job(JobType.EVALUATION, job_create)
+    assert job_ret is not None
+    jobs = lumi_client.jobs.get_jobs()
+    assert jobs is not None
```

Comment:
suggestion 
```
assert lumi_client.jobs.get_jobs() is not None
```

so the reader can skim all the asserts at once

Reply:
Ok